scripts/dev-environment-setup.sh

```diff
@@ -383,43 +389,29 @@ install_aws_cli () {
     fi
 }
 
-install_databases () {
-    SKIP="$(check_homebrew)"
+start_database() {
+  # Check if mongodb is installed
+  brew ls --versions mongodb > /dev/null
+  if [ $? -eq 0 ]; then
+    echo "Error: MongoDB is already installed via brew. Please uninstall it first."
+    exit 1
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** I think throwing an error here could be a bit heavy-handed. I think there are valid cases for having Mongo/Redis installed in Brew for use outside of the Novu project. Perhaps we can log a warning here instead?‏